model/src/pyrenew/latent/infections.py

```diff
@@ -18,11 +17,11 @@ class InfectionsSample(NamedTuple):
 
     Attributes
     ----------
-    infections : ArrayLike | None, optional
+    post_seed_infections : ArrayLike | None, optional
         The estimated latent infections. Defaults to None.
     """
 
-    infections: ArrayLike | None = None
+    post_seed_infections: ArrayLike | None = None
```

Comment:
Is it always the case that it will be *post* seed?

Reply:
Yes, the `sample` function requires the seed infections as an argument, so this is refactored to only return the post-seed infections.